src/Sentry.NLog/SentryTarget.cs

```diff
@@ -404,14 +404,14 @@ protected override void Write(LogEventInfo logEvent)
             if (logEvent.Level >= Options.MinimumBreadcrumbLevel)
             {
                 var breadcrumbFormatted = RenderLogEvent(BreadcrumbLayout, logEvent);
-                var breadcrumbCategory = RenderLogEvent(BreadcrumbCategory, logEvent);
+                string? breadcrumbCategory = RenderLogEvent(BreadcrumbCategory, logEvent);
```

Comment:
I dunno why this errors now

Reply:
It only errors in Rider because the IDE is not smart enough to infer that a type can be nullable.